build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -266,4 +266,11 @@
         <Method name="fromFile"/>
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE"/>
     </Match>
+
+
+    <!-- New flags as of spotbogs 4.7.3.5 -->
+    <!-- TODO: Fix or explicitly exclude each occurrence -->
```

Comment:
FYI I've created a back log item to track this because there are *a lot* of new findings (most of them can be ignored, but we need to evaluate each one).